### PR TITLE
Temp merge develop

### PR DESCRIPTION
**Pull request by kommiharibabu**

### CODE REVIEW

**Comment by kommiharibabu** [comment]
No major conflicts, it is a simple merge. New/rename files in durable history branch have old copyright notices. I raised another ticket to deal with those files copyright changes.

File: test/evergreen.yml [hunk 1]
```diff
@@ -1979,9 +1979,9 @@ tasks:
       - func: "format test script"
         vars:
           test_env_vars: ASAN_OPTIONS="detect_leaks=1:abort_on_error=1:disable_coredump=0" ASAN_SYMBOLIZER_PATH=/opt/mongodbtoolchain/v3/bin/llvm-symbolizer
-          # run for 24 hours ( 24 * 60 = 1440 minutes), don't stop at failed tests, use default config
-          format_test_script_args: -t 1440
-
+          # run for 24 hours ( 24 * 60 = 1440 minutes), skip known errors, don't stop at failed tests, use default config
+          format_test_script_args: -E -t 1440
+
```
Comment (quchenhao) [inline test/evergreen.yml]: Empty space?

Comment (kommiharibabu) [reply test/evergreen.yml]: Will fix it in the actual merge.

Comment (kommiharibabu) [reply test/evergreen.yml]: The white space is present in the `develop` branch itself, so I didn't fix it here.

**Pull request closed without merging**